docs/zq/language.md

```diff
@@ -1760,6 +1760,48 @@ produces
 {collect:[1,4,7]}
 {collect:[1,2,3]}
 ```
+
+### 8.2 Lateral Expressions
+
+Lateral subqueries can also appear in expression context using the
+parenthesized form:
+```
+( over <expr> [, <expr>...] [with <var>=<expr> [, ... <var>[=<expr>]] | <lateral> )
```

Comment:
Having different syntax for the operator and expression forms is surprising. I think we should unify them.

Reply:
Agreed this needs some more thought.  Let's merge and revisit soon.